pkg/transform/testdata/mdox4.yaml

```diff
@@ -0,0 +1,51 @@
+version: 1
+
+inputDir: "testdata/testproj"
+outputDir: "testdata/tmp/test4/4"
+extraInputGlobs:
+  - "testdata/test.md"
+  - "testdata/teststatic"
+
+glueLink: "https://github.com/bwplotka/mdox/tree/main"
```

Comment:
Let's rethink the name.. maybe `linkPrefixForNonMarkdownResources`

Reply:
Done!